src/settings/Settings.tsx

```diff
@@ -552,6 +552,14 @@ export const SETTINGS: { [setting: string]: ISetting } = {
         default: false,
         controller: new ReloadOnChangeController(),
     },
+    "feature_notifications": {
+        isFeature: true,
+        labsGroup: LabGroup.Messaging,
+        displayName: _td("labs|notifications"),
```

Comment:
I think `notifications` is a bit too generic here. What about `notifications_panel`?

Reply:
That's the current terminology we refer this piece of UI with. I'll defer to @daniellekirkwood for the final wording

Reply:
In translation the name is „Notifications panel“. If I see `notifications` as a translator I would naively translate it with „Notifications“ / „Benachrichtigungen“ instead of „Notifications panel“ / „Benachrichtigungsbereich“.